src/test/java/com/wikia/webdriver/testcases/globalshortcuts/ActionExplorerTest.java

```diff
@@ -0,0 +1,49 @@
+package com.wikia.webdriver.testcases.globalshortcuts;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.helpers.Browser;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.oasis.components.globalshortcuts.ActionExplorerModal;
+
+import org.testng.annotations.Test;
+
+@Execute(onWikia = "globalshortcuts-en")
+@InBrowser(browser = Browser.CHROME)
+public class ActionExplorerTest extends NewTestTemplate {
+
+  private ActionExplorerModal actionExplorerModal;
+
+  private void init() {
+    this.actionExplorerModal = new ActionExplorerModal(driver);
+
+    new Navigate(driver).toPage("/wiki/Globalshortcuts-en_Wikia");
+  }
+
+  @Test(groups = "globalShortcuts_actionExplorer_openAndCloseModalByShortcuts")
+  public void globalShortcuts_actionExplorer_openAndCloseModalByShortcuts() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)